spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/TraceKeys.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.sleuth.instrument;
+
+/**
+ * Well-known {@link org.springframework.cloud.sleuth.Span#tag(String, String) span tag} keys.
```

Comment:
@jcarres-mdsol cool. I did caveat the URL, but I'll be more explicit about it being optional.

Do you feel strongly about "http.uri" vs "http/path"? I feel the latter is more honest as we strip query params and there's nothing about "uri" that indicates that.